back/config/schemas/settings.schema.json.erb

```diff
@@ -901,30 +901,6 @@
         }
       },
 
-      "manual_tagging": {
```

Comment:
Do we need to manually remove this part of the schema for existing tenants that might have this persisted in the database already?

Reply:
Good question! The answer is no because a rake task is run to remove those (and adding missing ones) when deploying.

Reply:
See: https://github.com/CitizenLabDotCo/citizenlab-ee/blob/12a2aeeb11f5304482956181992dafdd1b4395fe/.circleci/config.yml#L331